tests/virt_autotest/login_console.pm

```diff
@@ -116,20 +116,6 @@ sub login_to_console {
 
     if (!get_var('UPGRADE_AFTER_REBOOT')) {
         set_var('REBOOT_AFTER_UPGRADE', '') if (get_var('REBOOT_AFTER_UPGRADE'));
-        if (is_xen_host) {
-            #send key 'up' to stop grub timer counting down, to be more robust to select xen
-            send_key 'up';
-            save_screenshot;
-
-            for (1 .. 20) {
-                if ($_ == 10) {
-                    reset_consoles;
-                    select_console 'sol', await_console => 0;
-                }
-                send_key 'down';
-                last if check_screen 'virttest-bootmenu-xen-kernel', 5;
-            }
-        }
```

Comment:
The login_console.pm is used by multiples tests out of VT MU .  I am afraid this can not be directly removed. Suggest to use some flag variable from testsuite settings to indicate whether you use grub2-set-default for xen selection or the original grub2 selection way.

Reply:
Agree. Please keep these statements on xen because most product tests rely on them to boot up from xen by choosing the entry in grub  during login rather than setting default boot in grub configuration presently.